gateway/1.5/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -34,22 +34,26 @@ As well as being able to configure additional Fluent Bit log output types, Flex
 
 == Before You Begin
 
-Before configuring Flex Gateway log output for Splunk, complete the following tasks:
+Before configuring Flex Gateway log output, complete the following tasks:
 
 * xref:flex-install.adoc[Download Flex Gateway]
 * xref:flex-{page-mode}-reg-run.adoc[Register and Run Flex Gateway]
 
-Additionally, depending on your output, ensure that the service where you are directing logs to is operational. For Splunk, you must configure:
-
-* A running https://www.splunk.com/[Splunk^] service. You can test Splunk by setting up a free Splunk Cloud trial.
-* Added an HTTP Event Collector (HEC) data input to the Cloud or Enterprise Splunk service. See the https://docs.splunk.com/Documentation/Splunk/7.0.3/Data/UsetheHTTPEventCollector[Splunk documentation^] about how to add a data input, and how to create an HEC token.
+Additionally, depending on your output, ensure that the service where you are directing logs to is operational and that you have collected the necessary information:
 
+* Splunk:
```

Comment:
I feel there is something missing in these bullets.
Is it possible to be consistent in their structure? Or is it possible to move this information to a table?